test/Handler/SitesSpec.hs

```diff
@@ -0,0 +1,90 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Handler.SitesSpec where
+
+import TestHelper hiding (createSite)
```

Comment:
I'm going to follow up with a more overall change to our helpers and
"factories". I'll definitely remove the hiding as part of that.
On Mar 6, 2015 7:14 AM, "Joe Ferris" <notifications@github.com> wrote:

> In test/Handler/SitesSpec.hs
> <https://github.com/thoughtbot/carnival/pull/149#discussion_r25950554>:
>
> > @@ -0,0 +1,90 @@
> > +{-# LANGUAGE OverloadedStrings #-}
> > +module Handler.SitesSpec where
> > +
> > +import TestHelper hiding (createSite)
>
> Should we try to rename this test helpers so that the names won't conflict?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/carnival/pull/149/files#r25950554>.
>